.github/workflows/documentation.yml

```diff
@@ -9,7 +9,7 @@ on:
 jobs:
   deploy:
     name: Deploy
-    runs-on: ubuntu-latest
+    runs-on: macOS-latest
```

Comment:
Documentation builds only JS, it is fine to stay on ubuntu.
Also this one does not have `Generate distributions` step from check.